tests/spec/w3c/headers-spec.js

```diff
@@ -105,6 +107,7 @@ describe("W3C — Headers", () => {
       const ops = makeStandardOps();
       const newProps = {
         specStatus: "REC",
+        group: "webapps",
```

Comment:
How about we default to `group: "webapps"` in makeStandardOps and override when needed?

Reply:
I thought of doing that... but I feel we should be further limiting `makeStandardOps()` because I can never remember what defaults it actually returns, which are probably out of date and can cause us to miss things: In the tests, I prefer we are explicit even if it means more repetition.